JiraQueryExecutor.py

```diff
@@ -1,99 +0,0 @@
-

```

Comment:
Please move the file with "git mv" before making change to it to retain the history and ease the review process (as you did for BearerAuth.py).

Reply:
Sorry will fix this. Thought I did.